05-UI-ReactJS/source/movieApp.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react"
+import {Router, Route, Link, IndexRoute, hashHistory, browserHistory} from "react-router"
+import {Provider} from "react-redux";
+import Movie from "./movie"
+import MoviesHandler from "./movieHandler";
+import MovieInput from "./movieInput";
+import store from "./store";
+
+class Application extends Component {
+  render () {
+    return (
+      <Provider store={store}>
+        <Router history={hashHistory}>
+          <Route path="/" component={Movie} />
```

Comment:
It would be good to have the movieInput and movieHandler paths nested with this root path.